bin/rucio-storage-consistency-actions

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Authors:
+# - Stefan Piperov <stefan.piperov@cern.ch>, 2021
+
+"""
+Storage-Consistency-Actions is a daemon to trigger the
+deletion of dark files and re-transfer missing files at
+sites, as identified by the Storage-Consistency-Scanner.
+"""
+
+import argparse
+import pathlib
+import signal
+
+from rucio.daemons.storage.consistency.actions import run, stop
+
+def get_parser():
+    """
+    Returns the argparse parser.
+    """
+    parser = argparse.ArgumentParser(description="The Consistency-Actions daemon is responsible for applying the corrective actions resulting from a consistency-check scan of an RSE.", epilog='''
+
+Run the daemon::
+  $ rucio-storage-consistency-actions --run-once --rses T2_US_Purdue T2_US_Nebraska --dark-threshold-percent 2.0 --miss-threshold-percent 1.5 --scanner-files-path /tmp/consistency-dump --sleep-time 10
+    ''')
+    parser.add_argument("--run-once", action="store_true", default=False, help='One iteration only')
+    parser.add_argument("--rses", nargs='+', default=[], type=str,
+        help='RSEs to check specified as a RSE expression, defaults to check (default: check all RSEs).')
+    parser.add_argument('--sleep-time', action="store", default=60, type=int,
+        help='Concurrency control: thread sleep time after each chunk of work')
+    parser.add_argument('--dark-min-age', action="store", default=28, type=int,
+        help='Min. age of a file to be considered as DARK.')
```

Comment:
can you add a comment which unit to use? seconds, days, etc...

Reply:
Agreed - will do! 
(good point!)